mentoring/mentoring.py

```diff
@@ -235,6 +279,44 @@ def next_step_url(self):
         return '/jump_to_id/{}'.format(self.next_step)
 
     @XBlock.json_handler
+    def get_results(self, queries, suffix=''):
+        results = []
+        if not self.show_extended_feedback():
```

Comment:
@Kelketek It's either a bad function name here, or a bad logic or a bad error message. It is possible to get results if extended feedback is not configured, but the thing is, it shouldn't be shown if it's not time to show it. This function probably needs a better name.